scenario-compiler/src/main/scala/pl/touk/nussknacker/engine/spel/SpelExpression.scala

```diff
@@ -389,4 +389,19 @@ object SpelExpressionParser extends LazyLogging {
     )
   }
 
+  private def createSpelParserConfiguration(classLoader: ClassLoader) = {
+    val autoGrowNullReferences  = false
```

Comment:
Let's move this values as a consts of SpelExpressionParser?

Reply:
I think they can be left as local values, since I extracted them only to avoid passing meaningless values to the constructor, like this:
```scala
    new SpelParserConfiguration(
      SpelCompilerMode.IMMEDIATE,
      classLoader,
      false,
      false,
      Integer.MAX_VALUE,
      Integer.MAX_VALUE
    )
```
All those extracted values are defaults in `SpelParserConfiguration` expect for `maximumExpressionLength`.